main/EDPftvarcon.F90

```diff
@@ -199,6 +199,16 @@ module EDPftvarcon
      real(r8), allocatable :: hydr_avuln_gs(:)      ! shape parameter for stomatal control of water vapor exiting leaf 
      real(r8), allocatable :: hydr_p50_gs(:)        ! water potential at 50% loss of stomatal conductance
 
+ !  PFT specific parameters for hydro dynamic roots 
```

Comment:
comments with descriptions of each parameter following their declarations please
include units as well if possible